nemo/utils/exp_manager.py

```diff
@@ -500,6 +521,16 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
             trainer.max_time = cfg.max_time_per_run
             trainer.callbacks.append(StatelessTimer(cfg.max_time_per_run))
 
+    if cfg.create_straggler_detection_callback:
+        if not HAVE_STRAGGLER_DET:
```

Comment:
@ericharper What is your preference when we don't have the package. Throw an error because it's an invalid config, or disable the FT features and try to proceed (printing a warning)?

Reply:
I think this is okay how it is.